test/testDelete.js

```diff
@@ -25,68 +25,75 @@ describe('routes', function () {
 
     describe("DELETE /api/issues/delete/all", () => {
         describe("One issue", () => {
-            it("status code 200, issue deleted", async () => {
-                await addIssue();
+            it("status code 200, issue deleted", (done) => {
```

Comment:
Both async and callback version are exactly the same. Async one is more modern and maybe more cleaner solution (no need for updates just fyi)